app/scripts/templates/settings/clients.mustache

```diff
@@ -18,7 +18,7 @@
         {{#clients}}
           <li class="client {{type}} client-{{clientType}} {{#isCurrentDevice}}client-current{{/isCurrentDevice}}"
             data-name="{{name}}" id="{{id}}">
-            <div class="client-name" title="{{name}}">{{name}}</div>
+            <div class="client-name" title="{{name}}" scope="{{scope}}">{{name}}</div>
```

Comment:
@divyabiyani this needs to be inside of the existing `title` attribute so we can see it on hover. 
Let's do `title="123Done - scopes: profile:write profile"`

Need to add a test to check that the `title` will display scopes. 

In addition this new scope display should be translated, so the template should check if `scope` is defined, if scope is available then 

You will need to have have something like this: `title={{#t}}%(email)s - scopes: %(scope)s{{/t}}`

See this example: https://github.com/mozilla/fxa-content-server/blob/master/app/scripts/templates/choose_what_to_sync.mustache#L2